### ISSUE TITLE
Current OmniGAN tasks

### ISSUE THREAD

**Issue by 51N84D** [cc-ai/kdb#140]
- [ ] Hyperparameters optimization for masker and painter respectively

- [ ] Try adding WD data (keep Unity as well)

- [ ] Learning rate decaying

- [ ] Curate and add new flood data for painter

- [ ] (Optional) Try ADVENT V2 i.e. intra-domain adaptation

**Comment by tianyu-z** [comment]
updated on Aug 10, 2020
What's Next:  
- [x] Multi-level Painter

- [ ] Load params from different checkpoints (Different heads converge with different speed)

- [ ] HRNet head in Masker? (I remember someone has done a lot of research on this.)

- [ ] Other suggestions from the Jury?

- [ ] A little suggestion here: GPT-3 API can be applied [here](https://forms.office.com/Pages/ResponsePage.aspx?id=VsqMpNrmTkioFJyEllK8sx3ELsv0PEhHphhNz30FttVUNkYwTlNPMVI1V0lXNjExMlExUlc4SE5YSS4u). I guess we maybe we can use it as a Question Answering robot on our website?
